spanner_orm/field.py

```diff
@@ -299,10 +321,14 @@ def field_type_from_ddl(ddl: str) -> FieldType:
     return Float()
   elif ddl == 'STRING(MAX)':
     return String()
+  elif (match := re.fullmatch(r'STRING\(((?:[0-9]+))\)', ddl)) is not None:
```

Comment:
(nit) `((?:...))` doesn't really make sense, because `?:` makes the inner group non-capturing but then the outer group captures it. Just use `(...)` for a capturing group without making an unnecessary non-capturing group right inside it. (Same comment below.)

Reply:
Done.